test/unit/ExchangeRates.js

```diff
@@ -37,3 +37,14 @@ describe("ExchangeRates", function () {
         );
     });
 });
+
+describe("ExchangeRate", function () {
+    it("fromBytes", function () {
+         const expirationTime = new Date("February 24, 2022 15:00:00 UTC");
+         const props = {
+             expirationTime: expirationTime,
+           }
+         const exchangeRate = new ExchangeRate(props);
+         expect(exchangeRate._toProtobuf().expirationTime.seconds).to.deep.equal(Long.fromValue({ low: 1645714800, high: 0, unsigned: false }));
```

Comment:
```suggestion
        const exchangeRate = ExchangeRate._fromProtobuf(new ExchangeRate({ expirationTime })._toProtobu());
        expect(exchangeRate.expirationTime).to.be.equal(expirationTime);
```
I think it's important to do both `ExchangeRate._[to|from]Protobuf()` to confirm it the round trip works as expected. I believe you comparing to `1645714800` is actually comparing to the wrong value according to the issue; unless this test was to confirm the we're producing the wrong value in which case you can ignore this sentence.

Reply:
I think you're right. I'll update the tests and recheck.